it/src/main/java/org/corfudb/universe/scenario/fixture/FixtureUtil.java

```diff
@@ -63,20 +64,25 @@
      * @param serverParamsBuilder corfu server builder with predefined parameters
      * @return list of VM corfu server params
      */
-    ImmutableList<CorfuServerParams> buildVmServers(
-            CorfuClusterParams cluster, VmCorfuServerParamsBuilder serverParamsBuilder,
-            String vmNamePrefix) {
+    ImmutableList<VmCorfuServerParams> buildVmServers(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/annymartinez.m/CorfuDB/pullRequest?prid=4663789)